docs/dev_guide/sunpy_stability.yaml

```diff
@@ -0,0 +1,54 @@
+cm:
+  status: mature
+  comments: Should see minor change in 1.0
+coordinates:
+  status: dev
+  comments: Should be stable but we reserve the right to change it for 1.0
```

Comment:
I would re-word this, to something like "should be stable, but more features are being added that might require backwards incompatible changes."